src/main/java/com/getbase/models/Source.java

```diff
@@ -13,6 +13,7 @@
   protected @JsonView(Views.ReadOnly.class) String createdAt;
   protected @JsonView(Views.ReadOnly.class) String updatedAt;
   protected @JsonView(Views.ReadWrite.class) String name;
+  protected @JsonView(Views.ReadOnly.class) String resourceType;
```

Comment:
This is not documented as read only. What about if some wants to create a source?